packages/flutter_tools/lib/src/commands/attach.dart

```diff
@@ -79,6 +88,7 @@ class AttachCommand extends FlutterCommand {
   @override
   final String description = 'Attach to a running application.';
 
+  // TODO(djshuckerow): this is now a confusing name, I would like to rename this.
   int get observatoryPort {
```

Comment:
@Hixie Is it a breaking change to rename a public field on a tool command?

Reply:
We shouldn't just remove them, but we could add a new command and mark the old ones as deprecated - with some runtime checking to ensure both aren't set

Reply:
I'm not too worried about the names of the arguments on the command line, just the name 'observatoryPort' here.

I've updated the TODO here with additional details on the value of a name change.